website/templates/project/statistics.mako

```diff
@@ -5,6 +5,23 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag"></div>
+<script src="/static/js/ads.js"></script>
+<script>
+  if( window.canRunAds === undefined ){
+    var banner = document.createElement('div');
```

Comment:
This is a little unwieldy, because building html in javascript is hard to read. However, the general idea seems good. Instead, make a CSS class called `.ad-hidden` or similar. Put the HTML in directly with the class of `.ad-hidden`. For the javascript, if `window.canRunAds` then change the style of .ad-hidden to `display: hidden`. Does that make sense?